lib/plaid.ex

```diff
@@ -77,16 +77,19 @@ defmodule Plaid do
   @spec make_request_with_cred(atom, String.t(), map, map, map, Keyword.t()) ::
           {:ok, HTTPoison.Response.t()} | {:error, HTTPoison.Error.t()}
   def make_request_with_cred(method, endpoint, cred, body \\ %{}, headers \\ %{}, options \\ []) do
+    {root_uri, cred} = Map.pop(cred, :root_uri)
+
+    re = "#{root_uri}#{endpoint}"
     rb = Map.merge(body, cred) |> Poison.encode!()
     rh = get_request_headers() |> Map.merge(headers) |> Map.to_list()
     options = httpoison_request_options() ++ options
-    request(method, endpoint, rb, rh, options)
-  end
 
-  def process_url(endpoint) do
-    require_root_uri() <> endpoint
+    request(method, re, rb, rh, options)
   end
 
+  def process_url("http" <> _rest = url), do: url
```

Comment:
Ideally this would be `"https"` but `Bypass` won't run over `https`. See PSPDFKit-labs/bypass#63 for more on that.